x-pack/plugins/ai_infra/llm_tasks/README.md

```diff
@@ -0,0 +1,45 @@
+# LLM Tasks plugin
```

Comment:
I'm wondering why this is it's own plugin instead of part of inference?

Reply:
To avoid monoliths-of-hell plugin patterns and the associated risks regarding cyclic dependencies problems, mostly. 

To elaborate a bit on the vision, I'm trying to keep the inference plugin as "thin" as possible. That way, consumers that want to just leverage the inference plugin to create their own tasks in their own package/plugin won't have to depend on all the dependencies from the "platform" tasks, that are in their own entity. That's the same idea being moving all inference type to the `inference-common` package: being able to depend on those types without depending on the runtime (the inference plugin)